appveyor.yml

```diff
@@ -20,22 +20,8 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      CONDA_PY: 27
 
     - TARGET_ARCH: x64
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 35
```

Comment:
Should be fixed in PR ( https://github.com/conda-forge/nilearn-feedstock/pull/9 ).